### ISSUE TITLE
request to add migration to storybook 8 using nx schematic generator in upcoming version

### ISSUE THREAD

**Issue by nehaGI** [nrwl/nx#27175]
### Current Behavior

nx 19.5.3 migrating storybook to version 7 not for storybook version 8.

### Expected Behavior

is nx planning to provide migration to storybook 7 to storybook 8 in upcoming version?

### Steps to Reproduce

1.install nx 19.5.3
2.  run storybook migration
   https://nx.dev/recipes/storybook/overview-angular

### Nx Report

```shell
Nx migration storybook to version 7. 

expectation - storybook migration from version 7 to version 8
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by effinrich** [comment]
Yes, please.  They're adding so many powerful testing tools, test generation, etc.
